src/fix_wall_reflect.h

```diff
@@ -28,10 +28,13 @@ class FixWallReflect : public Fix {
  public:
   FixWallReflect(class LAMMPS *, int, char **);
   virtual ~FixWallReflect();
+  virtual void wall_particle(int m, int which, double coord);
   int setmask();
   void init();
   void post_integrate();
-
+  enum{XLO=0,XHI=1,YLO=2,YHI=3,ZLO=4,ZHI=5};
```

Comment:
@akohlmey I don't think we put enums in header files?

Reply:
We didn't in the past, but we should. I have already submitted multiple pull requests that implemented increments of this change. The header is the right place and helps to keep settings consistent. We had multiple bugs because people modified an enum in the implementation of the base class, but not in the derived classes. They incur no storage cost and are inside the name space (unlike defines).